js/widgets/widget.theme.js

```diff
@@ -29,17 +29,30 @@
 
 $.mobile.widget.theme = {
 	_create: function() {
+		var that = this;
 		this._super();
 		if ( !this.options.enhanced && this.options.theme ) {
```

Comment:
Popups are hidden during enhancement so i'm not really sure how detaching would make a difference detaching. Either way it wont cause any sort of reflow or repaint until its shown?